services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -86,16 +92,21 @@ export const DashboardPage = ({ reportType }: Props) => {
     let newReportsToDisplay = reportsByState;
     if (!userIsAdmin) {
       newReportsToDisplay = reportsByState?.filter(
-        (report: ReportShape) => !report?.archived
+        (report: DashboardReportShape) => !report?.archived
       );
     }
     setReportsToDisplay(newReportsToDisplay);
   }, [reportsByState]);
 
-  const enterSelectedReport = async (report: ReportShape) => {
+  const enterSelectedReport = async (report: DashboardReportShape) => {
+    const reportKeys: ReportKeys = {
+      state: report.state,
+      id: report.id,
+    };
+    const foundReport: ReportShape = await fetchReport(reportKeys);
```

Comment:
'foundReport' refers to a report that the user entered, right? Not sure the naming reflects that. But it's a nit.

Reply:
Nah its a good call out. Any recommendations for what you would call it instead? Would `reportToEnter` make more sense?

Reply:
`selectedReport` 😀

Reply:
Maybe touchedReport?